src/test/java/com/kodilla/ecommercee/repository/ProductRepositoryTestSuite.java

```diff
@@ -0,0 +1,171 @@
+package com.kodilla.ecommercee.repository;
+
+import com.kodilla.ecommercee.domain.Group;
+import com.kodilla.ecommercee.domain.Product;
+import com.kodilla.ecommercee.exceptions.GroupNotFoundException;
+import com.kodilla.ecommercee.exceptions.ProductNotFoundException;
+import com.kodilla.ecommercee.service.GroupDbService;
+import com.kodilla.ecommercee.service.ProductDbService;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class ProductRepositoryTestSuite {
+
+    @Autowired
+    private ProductDbService productDbService;
+    @Autowired
+    private ProductRepository productRepository;
+    @Autowired
+    private GroupRepository groupRepository;
+    @Autowired
+    private GroupDbService groupDbService;
+
+    private Group clothes;
+    private Product trousers;
+    private Product boots;
+    private Product shirt;
+
+
+    private void createData() {
+        List<Product> productList = new ArrayList<>();
+        clothes = new Group(null, "clothes", productList);
+        trousers = new Product(null, "Trousers", "Denim", new BigDecimal(31), clothes);
+        boots = new Product(null, "Boots", "Leather", new BigDecimal("43.2"), clothes);
+        shirt = new Product(null, "shirt", "White", new BigDecimal("43.2"), clothes);
+        productList.add(trousers);
+        productList.add(boots);
+        productList.add(shirt);
+    }
+
+
+    @Test
+    public void testGetProduct() {
```

Comment:
```suggestion
    public void testSaveProduct() {
  ```
Testujesz zapisywanie produktów, nie pobieranie.